test/studies/shootout/submitted/fasta2.execopts

```diff
@@ -0,0 +1 @@
+--blockSize=4
```

Comment:
Without it, the correctness test is serialized which sometimes makes correctness errors in the parallel strategy not get caught until performance testing (if they're caught there).  So to the extent that the two versions might diverge from one another over time, I think it makes sense to correctness test it in this way, yes.  Let me know if you disagree.